metric_learn/mmc.py

```diff
@@ -22,13 +22,14 @@
 from sklearn.metrics import pairwise_distances
 from sklearn.utils.validation import check_array, check_X_y
 
-from .base_metric import BaseMetricLearner
-from .constraints import Constraints
+from .base_metric import PairsMetricLearner, SupervisedMetricLearner
+from .constraints import Constraints, ConstrainedDataset, unwrap_pairs, \
+  wrap_pairs
```

Comment:
Style nitpick: Use parens to break up long import lists:

```python
from .constraints import (
    Constraints, ConstrainedDataset, unwrap_pairs, wrap_pairs)
```

Reply:
Thanks !